doc/API.md

```diff
@@ -1,161 +1,8 @@
-| ```pause()```| | ```this```| stops the animation but keep the current time (play enables the animation again)
-| ```toggle()```| | ```this```| toggles (pause/play) the animation 
-| ```getStep()``` | | current animation step (integer)   | gets the current animation step
-| ```getTime()``` | | current animation time (Date) | gets the real animation time
-| ```isRunning()``` |  | `true`/`false` | describes whether the Torque layer is playing or is stopped 
-
-### Layer control methods
-
-| Method    | options    | returns   | Description                            |
-|-----------|:-----------|:----------|:---------------------------------------|
-| ```hide()``` | none    |```this```  | hides the Torque layer |
-| ```show()```| none| ```this``` | shows the Torque layer |
-
-### Style methods 
-
-| Method    | options    | returns   | Description                            |
-|-----------|:-----------|:----------|:---------------------------------------|
-| ```setCartoCSS(cartocss)``` | ```cartocss string```    | ```this```   | style the map rendering using client-side cartocss (not available with named maps) | 
-
-The full CartoCSS spec is not supported by Torque but instead only a limited subset with some additions related to torque rendering. To see the full list of supported parameters, read the [Torque CartoCSS documentation](CartoCSS.md). ``value`` and ``zoom`` variables can be used. ``value`` is the value of aggregation (see ``countby`` constructor option). ``zoom`` is the current zoom being rendered
-
-TorqueLayer currently expects ```marker``` styling.
-
-##### CartoCSS Example
-
-This should be ```string``` encoded in Javascript.
-
-```css
-#layer {
-  marker-fill: #662506;
-  marker-width: 20;
-  [value > 1] { marker-fill: #FEE391; }
-  [value > 2] { marker-fill: #FEC44F; }
-  [value > 3] { marker-fill: #FE9929; }
-  [value > 4] { marker-fill: #EC7014; }
-  [value > 5] { marker-fill: #CC4C02; }
-  [value > 6] { marker-fill: #993404; }
-  [value > 7] { marker-fill: #662506; }
-  [frame-offset = 1] {  marker-width: 20; marker-fill-opacity: 0.05;}' // renders the previous frame
-  [frame-offset = 2] {  marker-fill: red; marker-width: 30; marker-fill-opacity: 0.02;}' // renders two frames ago from the current being rendered
-}
-```
-
-### Data methods
-| Method    | options    | returns   | Description                            |
-|-----------|:-----------|:----------|:---------------------------------------|
-| ```setSQL(sql statement)``` | ```SQL string```    | ```this```   | Change the SQL on the data table (not available with named maps) | 
-| ```error(callback)``` | ```callback function with a list of errors as argument```    | ```this```   | specifies a callback function to run if there are query errors | 
-
-##### SQL Example
-
-Limit the data used in the Torque map.
-```js
-torqueLayer.setSQL("SELECT * FROM table LIMIT 100");
-```
-
-### Interaction methods (only available for Leaflet)
```

Comment:
Added tip that all these interaction methods are available for Google map layers, with the exception of `invalidate`. Check committed changes to see exact wording.